glide-core/redis-rs/redis/tests/test_cluster_scan.rs

```diff
@@ -857,4 +902,86 @@ mod test_cluster_scan_async {
             }
         }
     }
+
+    #[tokio::test]
+    #[serial_test::serial]
+    async fn test_async_cluster_scan_with_missing_node() {
```

Comment:
nit: 
```suggestion
    async fn test_async_cluster_scan_with_uncovered_slots() {
```

Reply:
also it isn't clear if this test tests for allowed uncovered slots or not , should be more described (in the test name and or in description), and I would expect to see two tests - one when allow_uncovered is false and second when it's true

Reply:
Its include both tests, ill make the flow in tests more clear. And there will be two more test, which are under work in a separate PR, since its not me that write them.